web/pgadmin/dashboard/static/js/Replication/index.jsx

```diff
@@ -0,0 +1,197 @@
+import { Box } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+
+import gettext from 'sources/gettext';
+import ReplicationSlotsSchema from './replication_slots.ui';
+import PgTable from 'sources/components/PgTable';
+import getApiInstance, { parseApiError } from '../../../../static/js/api_instance';
+import SectionContainer from '../components/SectionContainer';
+import ReplicationStatsSchema from './replication_stats.ui';
+import RefreshButton from '../components/RefreshButtons';
+import { getExpandCell } from '../../../../static/js/components/PgTable';
+import { usePgAdmin } from '../../../../static/js/BrowserComponent';
+import url_for from 'sources/url_for';
+import { getSwitchCell } from '../../../../static/js/components/PgTable';
```

Comment:
import { getExpandCell } from '../../../../static/js/components/PgTable';
import { getSwitchCell } from '../../../../static/js/components/PgTable';

Combine the import statements